TicTacToe/first.js

```diff
@@ -0,0 +1,116 @@
+let count = 1;
+let score1 = 0;
```

Comment:
```suggestion
let playerOne = 0;
```
Deves ter variaveis mais especificas